To: vim_dev@googlegroups.com
Subject: Patch 7.3.252
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.252 (after 7.3.247)
Problem:    Tests fail. (David Northfield)
Solution:   Add missing update for .ok file.
Files:      src/testdir/test81.ok


*** ../vim-7.3.251/src/testdir/test81.ok	2011-06-26 05:36:07.000000000 +0200
--- src/testdir/test81.ok	2011-07-15 13:14:27.000000000 +0200
***************
*** 1,4 ****
--- 1,6 ----
  aaa two
+     z
+ y
  bbb y
  ccc
  ddd yee y
*** ../vim-7.3.251/src/version.c	2011-07-15 17:51:30.000000000 +0200
--- src/version.c	2011-07-15 17:55:25.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     252,
  /**/

-- 
You cannot propel yourself forward by patting yourself on the back.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
